main/src/com/google/refine/expr/functions/EpochToDate.java

```diff
@@ -0,0 +1,66 @@
+
+package com.google.refine.expr.functions;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.Instant;
+import java.util.Properties;
+import com.google.refine.expr.EvalError;
+import com.google.refine.grel.ControlFunctionRegistry;
+import com.google.refine.grel.Function;
+
+public class EpochToDate implements Function {
+
+    @Override
+    public Object call(Properties bindings, Object[] args) {
+        if (args.length >= 1 && args[0] instanceof Number) {
+            long epoch = (long) args[0];
+            ZoneId zoneId = ZoneId.of("UTC");
+            Instant instant = null;
+            OffsetDateTime date = null;
+            if (args.length == 1) {
+                instant = Instant.ofEpochSecond(epoch);
+                date = OffsetDateTime.ofInstant(instant, zoneId);
+                return date;
+            } else if (args.length == 2 && args[1] instanceof String) {
+                Object o2 = args[1];
+                String type = ((String) o2).toLowerCase();
+                if (type.equals("second")) {
```

Comment:
Would 'unit' be a more intuitive name for this property rather than 'type'? Whichever is used in the code, IMO for the user 'unit' makes more sense